authority/voting/server/server.go

```diff
@@ -138,6 +142,35 @@ func (s *Server) listenWorker(l net.Listener) {
 		l.Close()
 		s.Done()
 	}()
+	for {
+		conn, err := l.Accept()
+		switch e := err.(type) {
+		case nil: // No Error
+		case net.Error:
+			if !e.Timeout() && !e.Temporary() {
+				s.log.Errorf("accept failure: %v", err)
+				return
+			}
+			continue
+		default:
+			s.log.Errorf("accept failure: %v", err)
+			return
+		}
+		s.Add(1)
+		s.onConn(conn)
+	}
+	// NOTREACHED
+}
+
+func (s *Server) listenWSWorker(l net.Listener) {
```

Comment:
does this work? if not we should remove it

Reply:
I believe that this code should work although there aren't tests or a websocket client as of yet.
I'll remove this function from this PR and put it in another PR.